src/views/home/home.vue

```diff
@@ -29,11 +29,15 @@ import VenueList from '@/components/venue-list/venue-list-component.vue';
 import AlertBanner from '@/components/alert-banner/alert-banner-component.vue';
 import Countdown from '@/components/countdown/countdown-component.vue';
 import ProgressBanner from '@/components/progress-banner/progress-banner-component.vue';
+import { VenuesService } from '../../services/venue-service';
```

Comment:
I noticed that the class name is VenuesService (plural) vs. the file name which is venue-service.ts (singular). Should the convention be that file name matches class name? Also since venue is our entity would it be better for the naming to singular?